src/scss/components/_modal-close-button.scss

```diff
@@ -43,7 +43,7 @@
 	}
 
 	&:hover {
-		.close-button__span {
+		.modal-close-button__span {
```

Comment:
don't use name of element in a class name

Reply:
Done